source/html/renderer/src/card-elements.ts

```diff
@@ -470,7 +470,7 @@ export class FactSet extends CardElement {
             return this.speak + '\n';
```

Comment:
Yes, I think there is a mistake in the doc. I'll discuss with the team, thanks for reporting.